resources/popularization/README.md

```diff
@@ -30,10 +28,20 @@ What is Angle Protocol?
 Angle Introduction Article
 {% endembed %}
 
-- _Details: A more thorough article to get a clear and thorough overview of the Angle Protocol and its different stakeholders_
+- _Details: A more thorough article to get a clearer overview of the Angle Protocol and its different stakeholders_
 - _DeFi Expertise Required: 2/5_
 - _Recommended If: You want to go deep in the subject, understand the vision behind the project and how it tries to achieve it_
 
+## Angle Explains Threads
+
+- _Details: regular threads about Angle or DeFi subjects related to the protocol. The goal is to give more insights to the Angle and wider crypto communities about those topics._
+- _DeFi Expertise Required: 2/5 to 4/5_
+- _Recommended if you want to dive deeper into various topics concerning Angle, DeFi, and stablecoins._
+
+**Threads**:
+- June 5th 2022: [veTokens and Liquid Lockers, Part 1](https://twitter.com/AngleProtocol/status/1533416014033059841)
```

Comment:
We can add a defi expertise note for each thread

Reply:
In the end I don't think so, I feel like it might throw off some people to see that it's too difficult and there is not much value to it